dynawo/cmake/FindSundials.cmake

```diff
@@ -27,28 +27,28 @@ if(NOT SUNDIALS_HOME AND NOT $ENV{SUNDIALS_INSTALL_DIR} STREQUAL "")
   set(SUNDIALS_HOME $ENV{SUNDIALS_INSTALL_DIR})
 endif()
 
-set(LIBRARY_DIR lib)
+set(SUNDIALS_LIBRARY_LOCATIONS ${SUNDIALS_HOME}/lib)
 if(MSVC)
-  set(LIBRARY_DIR bin)
+  list(APPEND SUNDIALS_LIBRARY_LOCATIONS ${SUNDIALS_HOME}/bin)
```

Comment:
it means that we install some things in lib on windows ?

Reply:
Currently, on Windows, FindSundials.cmake needs this path ${SUNDIALS_HOME}/bin for the install

But for the deploy, FindSundials.cmake needs both these paths ${SUNDIALS_HOME}/bin and ${SUNDIALS_HOME}/lib

Reply:
so in the deploy we put some things in lib ?

Reply:
Ok I think that's where the problem is. In install we do this also ? On Windows the good practice is to put all dlls in bin and not lib...

Reply:
I think I remember that Frederic had an issue and this is how he solved it, but I don't remember what the issue was exactly. Maybe we could try and check by ourselves if we can compile and run private repo with everything in bin

Reply:
> Ok I think that's where the problem is. In install we do this also ? On Windows the good practice is to put all dlls in bin and not lib...

In the install, every libraries are put in the bin folder. There is no lib folder in the Windows install for SUNDIALS.